lib/rules/import-notation/__tests__/index.js

```diff
@@ -0,0 +1,201 @@
+	],
+
+	reject: [
+		{
+			code: '@import foo.css;',
+			fixed: '@import url(foo.css);',
+			message: messages.expected('foo.css', 'url'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 16,
+		},
+		{
+			code: '@import "foo.css";',
+			fixed: '@import url("foo.css");',
+			message: messages.expected('foo.css', 'url'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 18,
+		},
+		{
+			code: "@import 'foo.css';",
+			fixed: "@import url('foo.css');",
+			message: messages.expected('foo.css', 'url'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 18,
+		},
+		{
+			code: "@import 'foo.css' print;",
+			fixed: "@import url('foo.css') print;",
+			message: messages.expected('foo.css', 'url'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 18,
+		},
+		{
+			code: "@import 'foo.css' supports(display: flex) screen and (max-width: 400px);",
+			fixed: "@import url('foo.css') supports(display: flex) screen and (max-width: 400px);",
+			message: messages.expected('foo.css', 'url'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 18,
+		},
+	],
+});
+
+testRule({
+	ruleName,
+	config: ['string'],
+	fix: true,
+
+	accept: [
+		{
+			code: '@import "foo.css";',
+		},
+		{
+			code: "@import 'foo.css';",
+		},
+		{
+			code: '@import foo.css;',
+		},
+		{
+			code: "@import 'common.css' screen;",
+		},
+		{
+			code: "@import 'foo.css' supports(display: flex) screen and (max-width: 400px);",
+		},
+	],
+
+	reject: [
+		{
+			code: '@import url("foo.css");',
+			fixed: '@import "foo.css";',
+			message: messages.expected('url("foo.css")', 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 23,
+		},
+		{
+			code: '@import uRl("foo.css");',
+			fixed: '@import "foo.css";',
+			message: messages.expected('uRl("foo.css")', 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 23,
+		},
+		{
+			code: '@import URL("foo.css");',
+			fixed: '@import "foo.css";',
+			message: messages.expected('URL("foo.css")', 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 23,
+		},
+		{
+			code: '@import url( "foo.css" );',
+			fixed: '@import "foo.css";',
+			message: messages.expected('url( "foo.css" )', 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 25,
+		},
+		{
+			code: "@import url('foo.css');",
+			fixed: "@import 'foo.css';",
+			message: messages.expected("url('foo.css')", 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 23,
+		},
+		{
+			code: "@import url( 'foo.css' );",
+			fixed: "@import 'foo.css';",
+			message: messages.expected("url( 'foo.css' )", 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 25,
+		},
+		{
+			code: '@import url(foo.css);',
+			fixed: '@import foo.css;',
+			message: messages.expected('url(foo.css)', 'string'),
+			line: 1,
+			column: 9,
+			endLine: 1,
+			endColumn: 21,
+		},
+		{
+			code: '@import url(foo.css) print;',
+			fixed: '@import foo.css print;',
```

Comment:
```suggestion
			fixed: '@import "foo.css" print;',
```

Let's default to double quotes (otherwise we're generating invalid CSS).